### PR TITLE
Changes for wip-13507 and test fixes

### PR DESCRIPTION
**Pull request by dzafman**
This is on top of another pull request wip-8885. I want to see what happens after that one merges.

### CODE REVIEW

File: tasks/scrub_test.py [hunk 1]
```diff
@@ -236,9 +236,8 @@ def _get_attrs(self, inc, attr_name):
             log.info('shard = %r' % shard)
             log.info('attr = %s' % attr_name)
             assert 'osd' in shard
-            assert attr_name in shard
             osd = shard['osd']
-            attr = shard[attr_name]
```
Comment (dzafman) [inline tasks/scrub_test.py]: @tchaikov Fix per your suggestion after moving to separate pull request.

File: tasks/scrub_test.py [hunk 2]
```diff
@@ -236,9 +236,8 @@ def _get_attrs(self, inc, attr_name):
...
+            attr = shard.get(attr_name, False)
```
Comment (dzafman) [inline tasks/scrub_test.py]: @tchaikov Fix per your suggestion after moving to separate pull request.

**Comment by dzafman** [comment]
Merge ceph/ceph-qa-suite#911 before this one.

**Comment by tchaikov** [comment]
lgtm

**Comment by athanatos** [comment]
lgtm
